main.go

```diff
@@ -1,18 +1,18 @@
 package main
```

Comment:
Bug: import statement for "github.com/birdsean/review-droid/github" was removed. Was this intentional or an oversight?